install/terraform/modules/gke/variables.tf

```diff
@@ -30,7 +30,8 @@ variable "cluster" {
   type        = map
 
   default = {
-    "zone"                    = "us-west1-c"
+    "location"		      = "us-west1-c"
+    "zone"                    = ""
```

Comment:
Should we remove zone from here to only have location in our sample configs?

Reply:
I think we can. There isn't really a reason to have zone here anymore.